extendr-api/src/matrix.rs

```diff
@@ -161,9 +161,10 @@ where
     fn from(array: RArray<T, [usize; 2]>) -> Self {
         let res = array.data.into();
         let dim = [array.dim[0] as i32, array.dim[1] as i32];
-        res.set_attrib(Symbol("dim"), dim);
-        res.set_attrib(Symbol("class"), "matrix");
-        res
+        res.set_attrib(dim_symbol(), dim)
+            .unwrap()
+            .set_attrib(class_symbol(), "matrix")
+            .unwrap()
```

Comment:
Here and throughout: is it a good idea to have all these `unwrap()` calls in the API? Can we somehow catch potential errors properly, or at least use `expect()`?

Reply:
Sounds good. These errors should never occur because the preconditions are met, but expect is better for locating regressions.